packages/runtime/src/components/builtin/Box/animations.tsx

```diff
@@ -1,365 +1,133 @@
+  const itemAnimationType = useMediaQuery(itemResponsiveAnimationType) || DEFAULT_ITEM_ANIMATE_TYPE
+  const duration = useMediaQuery(responsiveDuration) || DEFAULT_BOX_ANIMATE_DURATION
+  const delay = useMediaQuery(responisveDelay) || DEFAULT_BOX_ANIMATE_DELAY
+  const actualDelay = delay * 1000
+  const actualDuration = duration * 1000
+
+  const className = useStyle({
+    ...exitedBoxAnimationProperties[animationType],
+    transition: `all 0ms`,
+    '&.show': {
+      ...enteredBoxAnimationProperties[animationType],
+      transition: `transform ${actualDuration}ms cubic-bezier(0.16, 0.84, 0.44, 1) ${actualDelay}ms,filter ${actualDuration}ms cubic-bezier(0.16, 0.84, 0.44, 1) ${actualDelay}ms, opacity ${actualDuration}ms ease ${actualDelay}ms`,
+    },
+    '&:not(.show) > div > div': {
+      ...exitedBoxAnimationProperties[itemAnimationType],
+      transition: `all 0ms !important`,
+    },
+    '&.show > div > div': {
+      ...enteredBoxAnimationProperties[itemAnimationType],
     },
-    [boxControls, itemControls],
-  )
-
-  const playSequence = useCallback(() => {
-    boxControls.stop()
-    itemControls.stop()
-    boxControls.set('exited')
-    itemControls.set('exited')
-    boxControls.start('entered')
-    itemControls.start('entered')
-  }, [boxControls, itemControls])
+  })
 
-  useEffect(() => {
-    if (isBoxVisible) {
-      setSequence('entered')
-    }
-  }, [
-    elements
-      ?.map(e => e.key)
-      .sort()
-      .join(),
-    setSequence,
-  ])
+  if (reducedMotion) {
+    return ''
+  }
 
-  useEffect(() => {
-    if (isBoxVisible) {
-      playSequence()
-    }
-  }, [
-    isBoxVisible,
-    boxAnimateType,
-    boxAnimateDuration,
-    boxAnimateDelay,
-    itemAnimateType,
-    itemAnimateDuration,
-    itemAnimateDelay,
-    itemStaggerDuration,
-    reducedMotion,
-    playSequence,
-  ])
+  return className
+}
 
-  const boxVariant = boxAnimations[boxAnimateType]
-  const itemVariant = boxAnimations[itemAnimateType]
+export function useItemAnimation(
+  responsiveDuration: ResponsiveValue<number> | undefined,
+  responisveDelay: ResponsiveValue<number> | undefined,
+  responsiveStagger: ResponsiveValue<number> | undefined,
+  index: number,
+) {
+  const reducedMotion = usePrefersReducedMotion()
+  const duration = useMediaQuery(responsiveDuration) || DEFAULT_BOX_ANIMATE_DURATION
+  const delay = useMediaQuery(responisveDelay) || DEFAULT_BOX_ANIMATE_DELAY
+  const stagger = useMediaQuery(responsiveStagger) || DEFAULT_ITEM_STAGGER_DURATION
+  const delayFromStagger = responsiveStagger == null || index == null ? 0 : stagger * index
+  const actualDelay = (delay + delayFromStagger) * 1000
+  const actualDuration = duration * 1000
+
+  const className = useStyle({
+    transition: `transform ${actualDuration}ms cubic-bezier(0.16, 0.84, 0.44, 1) ${actualDelay}ms,filter ${actualDuration}ms cubic-bezier(0.16, 0.84, 0.44, 1) ${actualDelay}ms, opacity ${actualDuration}ms ease ${actualDelay}ms !important`,
```

Comment:
How come we're using "important" here? The `cx` and the order of the classes should take care of specificity and if the component gets its transition property overwritten, that should be ok

Reply:
You are right. This one slipped through. I'll double check both `!important` uses because I might have made the other one because I forgot I had added this one 😅